client/auth.go

```diff
@@ -5,11 +5,16 @@ import (
 	"crypto/tls"
 	"encoding/binary"
 
+	"fmt"
```

Comment:
remove the above blank line to group the standard libraries.

Reply:
removed